hs_odm2/management/commands/odm2-listall.py

```diff
@@ -0,0 +1,14 @@
+"""This prints all variable names matching a specific prefix
+
+* By default, prints errors on stdout.
+"""
+
+from django.core.management.base import BaseCommand
+from hs_odm2.models import ODM2Variable
+
+
+class Command(BaseCommand):
+    help = "query ODM2 variable name database for all entries."
+
+    def handle(self, *args, **options):
+        print(list(ODM2Variable.all()))
```

Comment:
I don't think this is very useful as a command?

Reply:
I didn't realize we were so stringent about Management Commands. Once I clear it with Alva, I'm happy to delete them all, however this does make Communities management more difficult. These are here as a convenience to Community managers, which may be hard to appreciate if you aren't in that role. Let's discuss and delete as necessary. My preference is to keep these around, until Communities is a more mature concept and these evolve more and eventually become parts of an admin dashboard.

Reply:
This is the first I've heard of a community manager, I don't know what that includes.  Generally management commands are run by programmers so I'm not sure how this is useful to a community manager.

Reply:
The point of this management command was to demo the use of the ODM2 database. If you don't want it, I will remove it. It was a convenience while I was on vacation.